android/build.gradle

```diff
@@ -10,17 +10,17 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.2'
-        classpath 'com.google.gms:google-services:4.3.3'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```

Comment:
Should we also upgrade the gradle-wrapper?

Reply:
AS hasn't suggested me to do that

Reply:
Looks like we're good because 4.2 will require gradle 7.